features/step_definitions/supplier_steps.rb

```diff
@@ -51,3 +51,7 @@
 Given /^that supplier has a service on the (.*) lot(?: for the (.*) role)?$/ do |lot_slug, role_type|
   @service = create_live_service(@framework['slug'], lot_slug, @supplier["id"], role_type)
 end
+
+Then 'that supplier visits their page for that service' do
+  page.visit("#{dm_frontend_domain}/suppliers/frameworks/#{@framework['slug']}/services/#{@service['id']}")
```

Comment:
It might make this more reusable if you dug the framework slug out of `@service` here instead of assuming that you've also got `@framework` set.